apps/site/assets/ts/stop/components/StopPage.tsx

```diff
@@ -71,13 +86,24 @@ export default ({
         <AlertsTab alertsTab={alertsTab} />
       ) : (
         <>
-          <Alerts alerts={highPriorityAlerts} />
-          <div className="container">
-            <h2>Station Information</h2>
-            <p>
-              See upcoming departures, maps, and other features at this
-              location.
-            </p>
+          <div
+            className={`m-stop-page__info-container${
+              isGlxOpen ? " glx-open" : ""
+            }`}
+          >
+            <div className="m-stop-page__info">
+              {highPriorityAlerts.length > 0 ? (
```

Comment:
any particular reason this is being recreated here instead of just using `<Alerts/>` component as before?

Reply:
The Alerts component comes wrapped in a `<div className="container">` , which is centered and contained using  margins that work fine on a white background but not with the green. 

![image](https://user-images.githubusercontent.com/2136286/159299507-1f633243-466e-4fe6-828b-64e37db99bde.png)

And I was hesitant to modify the `<Alerts />` component itself, since it's likely used in many places.

Going to try to import it again now that I've updated the surrounding styling anyway, will push up another commit soon

Reply:
@bfauble Updated this to just use <Alerts />.

Also modified some styling and checked it with a fake alert, it will look about this

![image](https://user-images.githubusercontent.com/2136286/159304506-d878aa2f-1245-4f16-a90a-87469d439d7f.png)